packages/react/react/src/hooks/useCombinedRefs.spec.tsx

```diff
@@ -1,28 +1,30 @@
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { useCallback, useRef } from 'react';
+import React, { useCallback, useRef } from 'react';
```

Comment:
I think you don't have to import `React` because `tsconfig.build.json` ignores `*.spec.tsx`. There is no problem to run test.

Reply:
I think fixing it like [this](https://github.com/toss/slash/pull/212) would be a better option.

Reply:
Yes I think so. `tsconfog.build.json` should be only used to build.

I was going to fix it but you already did.